pkg/apis/kops/componentconfig.go

```diff
@@ -164,6 +164,14 @@ type KubeletConfigSpec struct {
 	FailSwapOn *bool `json:"failSwapOn,omitempty" flag:"fail-swap-on"`
 	// ExperimentalAllowedUnsafeSysctls are passed to the kubelet config to whitelist allowable sysctls
 	ExperimentalAllowedUnsafeSysctls []string `json:"experimentalAllowedUnsafeSysctls,omitempty" flag:"experimental-allowed-unsafe-sysctls"`
+	// EventQps If > 0, limit event creations per second to this value. If 0, unlimited.
+	EventQPS *int32 `json:"eventQps,omitempty" flag:"event-qps"`
```

Comment:
So we now (finally) have more componentconfig upstream in kubernetes/kubernetes.  We should try to be consistent where it's possible to smooth the path for users.

Looking at componentconfig, the json form should probably be capitalized as `QPS`, and in fact should be `eventRecordQPS`

(`pkg/kubelet/apis/kubeletconfig/v1beta1/types.go` has `EventRecordQPS *int32 `json:"eventRecordQPS,omitempty"``